### ISSUE TITLE
Dollar sign ($) as first character throws error "Illegal group reference: group index is missing"

### ISSUE THREAD

**Issue by sgoetz-brox** [atextor/owl-cli#6]
When trying to use `owl write` with a Turtle file that gets a prefixed name’s local part that starts with `$`, an error is thrown.

The expected behaviour, I think, would be to escape it (as it’s not allowed to be a [start character](https://github.com/atextor/owl-cli/issues/4)):

    ex:\$10 a ex:Test .

----

`test.ttl`:

```
@prefix ex: <https://example.org/> .

<https://example.org/$10> a <https://example.org/Test> . # valid
```

`owl write test.ttl` output:

>     @prefix ex: <https://example.org/> .
>
> Error: Illegal group reference: group index is missing

**Comment by atextor** [comment]
Hi @sgoetz-brox,
thanks for reporting. This should indeed not happen.

**Comment by atextor** [comment]
Fixed in [v1.2.3](https://github.com/atextor/owl-cli/releases/tag/v1.2.3).
